Corrected ### and *** in German and English files

sonofmun opened the pull request.

Commit:
Corrected ### and *** in German and English files

ghost commented:
## Words

| Changed   |   Status |
|:----------|---------:|
| `eng`     |    -9253 |
| `ger`     |   -13613 |

*[Hook UI build recap](http://ci.perseids.org/repo/PerseusDL/canonical-farsiLit/3144)*

Commit:
Added <caesura/> between line segments

This also required that the texts be tested as TEI since <caesura/> is not allowed in Epidoc. So I made those changes as well.

ghost commented:
## Global

| Changed         |   Status |
|:----------------|---------:|
| `coverage`      |       20 |
| `texts_passing` |        3 |

## Words

| Changed   | Status   |
|:----------|:---------|
| `UNK`     | Deleted  |
| `eng`     | New      |
| `far`     | New      |
| `ger`     | New      |

## Units

| Changed                                           | Status   |
|:--------------------------------------------------|:---------|
| `./data/hafez/divan/hafez.divan.perseus-eng1.xml` | Passing  |
| `./data/hafez/divan/hafez.divan.perseus-far1.xml` | Passing  |
| `./data/hafez/divan/hafez.divan.perseus-ger1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/PerseusDL/canonical-farsiLit/3146)*
